pages/legal/privacy.vue

```diff
@@ -144,7 +144,7 @@
       </p>
 
       <p>
-        Modrinth makes use external ads providers. These include BuySellAds. You
+        Modrinth makes use of external ads providers. These include BuySellAds. You
```

Comment:
Lint failed

Reply:
Well...
```
Error:   1[47](https://github.com/modrinth/knossos/runs/8274622938?check_suite_focus=true#step:7:48):80  error  Delete `·You`  prettier/prettier
Error:   148:8   error  Insert `·You`  prettier/prettier
```
Not sure what this should tell me...

Reply:
The "You" needs to be on the next line

Reply:
fixed